v2/cache.go

```diff
@@ -13,10 +13,10 @@
 package cache
 
 import (
-	"container/list"
 	"fmt"
-	"sync"
 	"time"
+
+	v3 "github.com/go-pkgz/expirable-cache/v3"
```

Comment:
I don't think this is a good idea. First of all, such a dependency feels unnatural and confusing; second - it will bring the v3 package as well as one wanted to include v2. And with vendored deps it will be even more confusing. My vote is to keep them separate